### PR TITLE
chore: adapt orogen to use helper functions from radar_base

### PR DESCRIPTION
**Pull request by kapeps**
Depends on:
- [ ] https://github.com/tidewise/drivers-radar_base/pull/9

# What is this PR for

Change the radar orogen to use helper functions

# How I did it

# Results, How I tested

Before the changes:
image

After the changes:
image

# Checklist
- [ ] Assign yourself  in the PR
- [ ] Write unit tests (when relevant)
- [ ] Run rubocop and rake tests locally
- [ ] If this PR initialize a CMAKE package please [enable styling and linting](https://github.com/tidewise/wetpaint-buildconf/blob/master/README.md#enabling-styling-and-linting-checks-for-a-cmake-package)
- [ ] Analyse if this PR modifies the UI, if so:
    - [ ] Tested Tupan's simulation (Charts) :world_map:
    - [ ] Tested Tupan's simulation (Hud) :joystick:
    - [ ] Tested ROV's simulation :joystick:

### CODE REVIEW

**Comment by wvmcastro** [comment]
Also, I think a better test would be to take the same "frame" rendered by current and previous implementation

File: tasks/EchoesToFrameConverterTask.cpp [hunk 1]
```diff
@@ -51,12 +52,16 @@ void EchoesToFrameConverterTask::updateHook()
 
     Radar radar_echo;
     while (_echo.read(radar_echo, false) == RTT::NewData) {
-        if (m_current_sweep_size != radar_echo.sweep_length ||
-            m_current_num_angles != (2 * M_PI / abs(radar_echo.step_angle.getRad())) ||
+        if (!m_lut->hasMatchingConfiguration(m_current_num_angles,
```
[the remark this answers is not in the document]
Comment (wvmcastro) [reply tasks/EchoesToFrameConverterTask.cpp:55]: Now I see that it is because the LUT does not have an default constructor

**Review by wvmcastro — APPROVED** [review]

File: tasks/EchoesToFrameConverterTask.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (kapeps) [reply tasks/EchoesToFrameConverterTask.cpp:57]: You're right, it shouldn't, it should only reset m_echoes

**Review by wvmcastro — APPROVED** [review]

**Review by jhonasiv — APPROVED** [review]

File: tasks/EchoesToFrameConverterTask.cpp [hunk 2]
```diff
@@ -96,38 +101,14 @@ void EchoesToFrameConverterTask::addEchoesToFrame(Radar const& echo,
...
 {
     m_cv_frame = 0;
     LOG_INFO_S << "Creating a frame...";
-    for (long i = 0; i < static_cast<long>(m_echoes.size()); i++) {
-        m_lut->updateImage(m_cv_frame,
-            i / m_current_sweep_size,
-            i % m_current_sweep_size,
-            m_echoes.at(i));
-    }
+    (*m_lut).drawImageFromEchoes(m_echoes, m_cv_frame);
```
Comment (jhonasiv) [inline tasks/EchoesToFrameConverterTask.cpp:111]:
```suggestion
    m_lut->drawImageFromEchoes(m_echoes, m_cv_frame);
```

**Pull request merged**